cmd/gRPC.go

```diff
@@ -0,0 +1,63 @@
+package cmd
```

Comment:
For the name of file is better to keep it all lowercase: "grpc.go"

Reply:
sure moved it to util/grpc.go